qstack/spahm/LB2020guess.py

```diff
@@ -290,8 +290,13 @@ def use_charge(self, mol):
 
   def use_ecp(self, mol, acbasis):
       acbasis = copy.deepcopy(acbasis)
+      q_cleaned = set()
       for iat, z in enumerate(mol.atom_charges()):
           q = mol.atom_pure_symbol(iat)
+          if q in q_cleaned:
+              continue
+          else:
+              q_cleaned.add(q)
```

Comment:
where is `q_cleaned` used ?

Reply:
what do you mean sorry?

Reply:
my bad, all good :ok_hand:

Reply:
if `q` is already in `q_cleaned` then we dont remove core electrons